tools/docs/config.ts

```diff
@@ -15,12 +15,16 @@ function genTable(obj: [string, string][], type: string, def: any): string {
     'default',
     'stage',
     'allowString',
-    'cli',
-    'env',
     'admin',
   ];
   obj.forEach(([key, val]) => {
     const el = [key, val];
+    if (key === 'cli' && !val) {
+      ignoredKeys.push('cli');
+    }
+    if (key === 'env' && !val) {
+      ignoredKeys.push('env');
+    }
```

Comment:
Doesn't this mean that no CLI or ENV values are added anymore, once the first invalid value has been encountered? Is that really what we want?

Reply:
🤔 need to check this in more detail

Reply:
My recommendation would be to drop this new piece of code here, and instead do the filtering while setting the property.

Reply:
@astellingwerf This ignoredKeys is for every config property. so unless there's more than 1 cli key or more than 1 env key for every config property that's not the case.

regarding your recommendation, do you think i should not add el.cli at all in case it doesnt exist? and no need for this check then? hmmm ill try that.

Reply:
If i do that, then i will have :
<html>
<body>
<!--StartFragment-->

cli | false
-- | --
env | false

<!--EndFragment-->
</body>
</html>

because i removed the ignoredKeys.. i will still have to ignore the cli/env params somehow, and shouldnt just print false false in the table